django/apps/registry.py

```diff
@@ -260,6 +260,27 @@ def get_registered_model(self, app_label, model_name):
                 "Model '%s.%s' not registered." % (app_label, model_name))
         return model
 
+    # This method is performance-critical when it comes to migrations.
+    # Since the swappable settings don't change after Django has loaded the
+    # settings, there is no reason to get the respective settings attribute
+    # over and over again.
+    @lru_cache.lru_cache(maxsize=None)
+    def get_swappable_settings_name(self, to_string):
+        """
+        For a given model string (e.g. "auth.User"), returns the name of the
+        corresponding settings name if it refers to a swappable model. If the
+        referred model is not swappable returns None.
+        """
+        for model in self.get_models(include_swapped=True):
+            swapped = model._meta.swapped
+            if swapped and swapped == to_string:
+                # If this model is swapped out for the model given by to_string
```

Comment:
I'd put the comment above the if statement (same below).
`# Is this model swapped out for the model given by to_string?`